lane.py

```diff
@@ -689,15 +691,18 @@ def works(self, _db, facets=None, pagination=None):
             MaterializedWork,
             MaterializedWorkWithGenre,
         )
-        if self.genre_ids:
+        if self.genre_ids or True:
```

Comment:
I'll get rid of this but it's pretty convincing evidence that we don't need MaterializedWork anymore.